app/Project.swift

```diff
@@ -26,7 +26,11 @@ func inspectBuildPostAction(target: TargetReference) -> ExecutionAction {
     .executionAction(
         title: "Inspect build",
         scriptText: """
-        eval "$($HOME/.local/bin/mise activate -C $SRCROOT bash --shims)"
+        if [ -f "$HOME/.local/bin/mise" ]; then
+            eval "$($HOME/.local/bin/mise activate -C $SRCROOT bash --shims)"
+        elif [ -f "$HOME/.local/share/mise/bin/mise" ]; then
+            eval "$($HOME/.local/share/mise/bin/mise activate -C $SRCROOT bash --shims)"
+        fi
```

Comment:
I don't understand why mise has a different path on the CI vs locally. Maybe someday I will explore why this is, but I don't have the energy right now